exercises/sum-of-multiples/src/test/java/SumOfMultiplesTest.java

```diff
@@ -162,11 +162,40 @@ public void testSumOfMultiplesOfOneUpToOneHundred() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testSumOfMultiplesOfNoneUpToTenThousand() {
+    public void testNoFactorsMeanAnEmptySum() {
 
         int[] set = {};
         int output = new SumOfMultiples(10000, set).getSum();
         assertEquals(0, output);
 
     }
+
+    @Ignore("Remove to run test")
+    @Test
+    public void testSumOfMultiplesOfZeroIsZero() {
+
+        int[] set = {
+            0
+        };
+        int output = new SumOfMultiples(1, set).getSum();
+        assertEquals(0, output);
+
+    }
```

Comment:
It looks like there's now been added another test after this one `the factor 0 does not affect the sum of multiples of other factors` and the version number has been bumped to 1.5.0. It's up to you if you want to add that as part of this PR or not! 🙂

Reply:
I will add this new test and update to version 1.5.0